_topics/en/cve.md

```diff
@@ -0,0 +1,176 @@
+      summary: >
+        Vulnerability in LND due to accepting non-standard "high-S"
+        signatures, which could've prevented channels from closing in a
+        timely manner and so allowed funds to be stolen.  Fixed in LND
+        0.10.0-beta
+
+    CVE-2018-17145:
+      link: /en/newsletters/2020/09/16/#inventory-out-of-memory-denial-of-service-attack-invdos
+      summary: >
+        Flooding a node with an excess of `inv` messages could lead to
+        the node running out of memory and crashing, increasing the risk
+        of [eclipse attacks][topic eclipse attacks] that can steal
+        money.  Affected Bitcoin Core, btcd, bcoin, and multiple
+        altcoin nodes.  Fixed in Bitcoin Core 0.16.2, Btcd 0.21.0-beta,
+        and Bcoin 1.0.2.
+
+    CVE-2017-12842:
+      link: /en/newsletters/2018/12/28/#cve-2017-12842
+      summary: >
+        Allows creating an SPV proof for a transaction that doesn’t
+        exist by specially crafting a real 64-byte transaction that gets
+        confirmed in a block.  Can be used to steal from SPV clients,
+        although other known attacks against SPV clients are cheaper.
+        Proposed to be fixed in the [consensus cleanup][topic consensus
+        cleanup] soft fork.
+
+    CVE-2017-18350:
+      link: /en/newsletters/2019/11/13/#cve-2017-18350-socks-proxy-vulnerability
+      summary: >
+        A vulnerability in Bitcoin Core affecting users of SOCKS
+        proxies.  Fixed in Bitcoin Core 0.16.0.
+
+    CVE-2013-2292:
+      link: https://bitcointalk.org/?topic=140078
+      summary: >
+        A vulnerability in Bitcoin Core that it was believed could've
+        allowed an attacker to create a block that would take over three
+        minutes to verify.  Proposed to be fixed in the [consensus
+        cleanup][topic consensus cleanup] soft fork, although the
+        suggestion solution encountered controversy.
+
+    CVE-2012-2459:
+      link: https://bitcointalk.org/?topic=102395
+      summary: >
+        It's possible to mutate a valid block into an invalid block that
+        commits to the same merkle root.  When vulnerable versions of
+        Bitcoin Core saw such an invalid block, they cached its
+        rejection and so would later reject the valid version of the
+        block, which could easily lead to short-term network partitions
+        where users could be tricked into accepting invalid bitcoins.
+        Fixed in Bitcoin 0.6.1.
+
+---
+Some CVEs that have been frequently mentioned by Optech have their own
+pages in the Optech topic index (see this page's *see also* section for
+links).  For any other CVEs affecting Bitcoin and Lightning, this
+catch-all page collects any significant mentions of those CVEs in Optech
+content.  This pages's CVE's include:
+
+{% for alias in page.aliases %}
+  * **[{{alias}}]({{page.extra.cves[alias].link | default: "#CVE-LINK-NOT-PROVIDED"}}):** {{page.extra.cves[alias].summary}}
```

Comment:
Right now CVE-2021-31876's entry resolves to link not provided, perhaps it should just go to /en/newsletters/2021/05/12/#cve-2021-31876-discrepancy-between-bip125-and-bitcoin-core-implementation